acceptance/tests/external_facts/structured_powershell_facts.rb

```diff
@@ -0,0 +1,80 @@
+# This test is intended to demonstrate that powershell external facts can return
+# YAML or JSON data.
+test_name "powershell facts can return structured data" do
```

Comment:
Has this been run through an ad-hoc pipeline or anything?

Reply:
The acceptance test seems a little redundant, the unit tests cover the same behavior.

Reply:
FACT-1653 asked for acceptance tests as part of its acceptance criteria, but I think I agree - let's see what @branan thinks - this did pass in an ad-hoc run a while ago.

Reply:
I think I'm in agreement that we want to just test this at the "unit"\* level - the acceptance test seems redundant to me.

* It's really an integration test, but whatever :)